GVRf/Framework/jni/objects/hybrid_object_jni.cpp

```diff
@@ -28,7 +28,9 @@ Java_org_gearvrf_NativeHybridObject_delete(JNIEnv * env,
 JNIEXPORT void JNICALL
 Java_org_gearvrf_NativeHybridObject_delete(JNIEnv * env,
         jobject obj, jlong jhybrid_object) {
-    delete reinterpret_cast<HybridObject*>(jhybrid_object);
+    HybridObject* hybrid_object = reinterpret_cast<HybridObject*>(jhybrid_object);
+    delete hybrid_object;
+    hybrid_object = nullptr;
```

Comment:
Thanks Chao. I commited the updated changes to fix it plus reverted hybrid_object_jni.cpp.

I still have some questions. 

1) How should we set native pointer to null to avoid native code from mistakenly referencing deleted pointers? Cause I think right now, doing checks for nullptr in native code are non effective.

2) I'm a little confused with GVRHybridObject. There's a mNativePointer on GVRHybridObject and another copy of it in GVRReference. The GVRHybridObject initialized both references upon construction.

mNativePointer = nativePointer;
sReferenceSet.add(new GVRReference(this, nativePointer, cleanupHandlers));

When closing an object the GVRReference native pointer is deleted and set to null but shouldn't GVRHybridObject native pointer also be set to null?